openscap_daemon/dbus_daemon.py

```diff
@@ -363,7 +363,7 @@ def containers(self):
     @staticmethod
     def _parse_only_cache(config, onlycache):
         if onlycache == 2:
-            return not config.fetch_cve
+            return config.fetch_cve
```

Comment:
Are you sure about this change?

`fetch_cve` means whether we should fetch CVEs, so IMO it is the opposite to `only_cache`. If we don't `fetch_cve` we use only cache.